### PR TITLE
refactor: bump packages and fix svga

### PR DESCRIPTION
**Pull request by 3rdvision**
| - | - |
| --- | --- |
| Issue | ripe-components-react-native-storybook has been collecting dust for about 7 months. It needs a bump in the packages and svga support that has been recently added to ripe-components-react-native. |
| Decisions |  - Bump packages and migration dependencies to react-native 0.68 <br> - Add missing support for svga assets. <br> - Fix pretty |

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -44,15 +44,15 @@
...
+        "@react-native-picker/picker": "^2.4.1",
         "@sentry/react-native": "^3.3.2",
         "buffer": "^6.0.3",
         "fast-text-encoding": "^1.0.3",
         "grapheme-splitter": "^1.0.4",
         "js-base64": "^3.7.2",
         "pusher-js": "^7.0.6",
         "react": "^17.0.2",
-        "react-native": "^0.67.3",
+        "react-native": "^0.68.3",
```
Comment (NFSS10) [inline package.json:55]: Bumping react-native versions is usually a risky move. I will mark this PR as risky because of this dependency bump

**Review by NFSS10 — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (3rdvision) [reply package.json:55]: We already moved to 0.68.3 in other react-native repos. This repo has been forgotten and broken for some time so I'm bumping it to the current versions and making it work. I think you can remove risky for this case.

File: package.json [hunk 2]
```diff
@@ -44,15 +44,15 @@
         "@react-native-community/datetimepicker": "^5.1.0",
         "@react-native-community/masked-view": "^0.1.11",
         "@react-native-community/netinfo": "^8.1.0",
-        "@react-native-picker/picker": "^2.3.1",
+        "@react-native-picker/picker": "^2.4.1",
```
Comment (beneditomauro) [inline package.json]: Just to be in pair with [react-components](https://github.com/ripe-tech/ripe-components-react-native/blob/6c22a438678617c64f8778d9d28b2f5a5e60e21d/package.json#L70) and [robin](https://github.com/ripe-tech/ripe-robin-revamp/blob/6360f1bdbce94d84e41d0ce48c9b4bb5346b2591/package.json#L44):
```suggestion
        "@react-native-picker/picker": "^2.4.4",

```

Comment (3rdvision) [reply package.json]: nice catch

**Review by beneditomauro — APPROVED** [review]

File: metro.config.js [hunk 3]
```diff
@@ -1,10 +1,20 @@
...
+                    experimentalImportSupport: false,
+                    inlineRequires: true
+                }
+            })
+        },
+        resolver: {
+            assetExts: [...assetExts, "svga"]
+        }
+    };
+})();
```
Comment (joao-conde) [inline metro.config.js]: Lets move the function out of export, and then export it

File: .eslintignore [hunk 4]
```diff
@@ -1,4 +1,6 @@
 /dist
 /temp
+/node_modules
 **/dist
+**/node_modules
```
Comment (joao-conde) [inline .eslintignore:5]: Doesnt the second ignore cover the first too?

**Review by joao-conde — CHANGES_REQUESTED** [review]

File: .eslintignore (hunk 4 above)
Comment (3rdvision) [reply .eslintignore:5]: Yes, I just double checked it it but we have it formatted this way in every repo so I left it this way

**Review by joao-conde — APPROVED** [review]

**Review by joamag — APPROVED** [review]

**Pull request merged**